apps/envited-x-data-space/common/api/api.ts

```diff
@@ -1,5 +1,8 @@
+import { httpGet } from '../http'
+
 export const fetchAssetDataByCID = async (cid: string) => {
-  const response = await fetch(`/api/assets/${cid}`)
+  // const response = await fetch(`/api/assets/${cid}`)
+  const asset = await httpGet(`/api/assets/${cid}`)
 
-  return response.json()
+  return asset
```

Comment:
This is the same as `return httpGet('...')` which allows you to write it as `... => httpGet('...')`